### ISSUE TITLE
v5.2.1 and v5.1.0 error - undefined object _NativeBridge

### ISSUE THREAD

**Issue by kilirobbs** [idehub/react-native-google-analytics-bridge#154]
expo: 19.0.0
react-native: 0.46.1 via (https://github.com/expo/react-native/archive/sdk-19.0.0.tar.gz)
react: 16.0.0-alpha.12
react-native-google-analytics-bridge: 5.2.1

My app has not been ejected from expo.  Followed the instructions multiple times, even created a new project, but no luck keep getting the following error

> undefined is not an object (evaluating '_NativeBridges.GoogleAnalyticsBridge.trackScreenView')
> 
> trackScreenView
>     GoogleAnalyticsTracker.js:56:26
> App
>     App.js:60:32
> <unknown>
>     ReactNativeStack-dev.js:1694:33
> measureLifeCyclePerf
>     ReactNativeStack-dev.js:1625:15
> _constructComponent
>     ReactNativeStack-dev.js:1686:56 ...

**Comment by cbrevik** [comment]
>My app has not been ejected from expo.

Expo.io has some [docs](https://docs.expo.io/versions/v19.0.0/introduction/faq.html#what-is-the-difference-between-expo-and-react-native) explaining the differences between Expo and React Native. At the bottom of that section you see:
>The most limiting thing about Expo is that you can’t add in your own native modules without detaching and using ExpoKit.

TLDR; You have to detach/eject in order to us **any** native modules not implemented by Expo.

**Comment by kilirobbs** [comment]
@cbrevik - thanks for quick answer and pointing me towards the docs.
